src/Method/Definition.php

```diff
@@ -172,43 +138,30 @@ public function setPrototypes(array $prototypes)
     /**
      * Get all prototypes
      *
-     * @return array $prototypes Array of \Laminas\Server\Method\Prototype objects or arrays
+     * @return Prototype[] $prototypes
      */
-    public function getPrototypes()
+    public function getPrototypes(): array
     {
         return $this->prototypes;
     }
 
-    /**
-     * Set method help
-     *
-     * @param  string $methodHelp
-     * @return \Laminas\Server\Method\Definition
-     */
-    public function setMethodHelp($methodHelp)
+    public function setMethodHelp(string $methodHelp): self
     {
-        $this->methodHelp = (string) $methodHelp;
+        $this->methodHelp = $methodHelp;
         return $this;
     }
 
-    /**
-     * Get method help
-     *
-     * @return string
-     */
-    public function getMethodHelp()
+    public function getMethodHelp(): string
     {
         return $this->methodHelp;
     }
 
     /**
      * Set object to use with method calls
      *
-     * @param  object $object
      * @throws Server\Exception\InvalidArgumentException
-     * @return \Laminas\Server\Method\Definition
      */
-    public function setObject($object)
+    public function setObject(object $object): self
```

Comment:
`null` need to be allowed since the `getObject()` allow null returns value.

Reply:
Getter return type must be nullable, since property might be uninitialized. "Resetting" an object to null makes only sense, if a previously set object needs to be reverted. Is there a use case for that?

Reply:
ok 👍 , make sense